custom/icds_reports/tasks.py

```diff
@@ -10,7 +10,7 @@
 celery_task_logger = logging.getLogger('celery.task')
 
 
-@periodic_task(run_every=crontab(minute=0, hour=0), acks_late=True, queue='background_queue')
+@periodic_task(run_every=crontab(minute=0, hour=12), acks_late=True, queue='background_queue')
```

Comment:
The version of master you branched off didn't have the stickler config in it. You should always try branch off latest master.